test/awap.jl

```diff
@@ -11,8 +11,9 @@
         joinpath(ENV["RASTERDATASOURCES_PATH"], "AWAP", "vprp", "vprph09", "20010101.grid.Z")
     @test zipname(AWAP, VapourPressure{H09}, Date(2001, 1)) == "20010101.grid.Z"
 
-    dates = DateTime(2001, 01, 01), DateTime(2001, 01, 02)
-    download_raster(AWAP, VapourPressure{H09}; dates=dates)
-
-    @test isfile(raster_file)
+    if Sys.islinux()
```

Comment:
@jamesmaino AWAP .Z files are a pretty obscure and ancient unix format. I'm not sure if you can get the `uncompress` command, or work out some way to unzip them from the command line then we can add a windows specific command for it. Currently it will probably fail on windows.

Reply:
Right... I use 7zip. For example, this works from the command line. Would this be easy enough to add?
 
```
7z e C:\RasterData\AWAP\temperature\minave\20181231.grid.Z
```